### DISCUSSION TITLE
Need to know the username and password after using default iso image

### DISCUSSION THREAD

**Discussion by SSWAMIN1SSS** [kairos-io/kairos#1921] [Q&A] [answered]
Hi All

Followed the steps to install Fedora ISO Image on a Client Device , the installation was successful but couldnt able to get the username and password for doing an SSH action.
Followed this steps
https://kairos.io/docs/reference/build-from-scratch/

Could anyone help me in knowing what is the default username and password to get into the machine.

**Comment by mudler** [comment, marked as answer]
Kairos needs a configuration file. Unless you are using the interactive installation, you must setup the user in the configuration file. See also the getting started here: https://kairos.io/docs/getting-started/#configuration
